src/main/java/core/basesyntax/service/CsvFileWriterService.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service;
+
+import core.basesyntax.FruitBatch;
+import core.basesyntax.FruitStorage;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CsvFileWriterService implements FileWriterService {
+
+    @Override
+    public void writeToFile(String filePath, Map<FruitBatch, Integer> fruits) {
+        String newFilePath = filePath.substring(0, filePath.length() - 4) + "_output.csv";
+        Map<String, Integer> stockOverview = formatData(fruits);
+        StringBuilder formattedLines = new StringBuilder("fruit,quantity\n");
+        for (Map.Entry<String, Integer> entry : stockOverview.entrySet()) {
+            formattedLines.append(entry.getKey())
+                    .append(",")
+                    .append(entry.getValue().toString())
+                    .append("\n");
+        }
+        PrintWriter writer = null;
+        try {
+            writer = new PrintWriter(new File(newFilePath));
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+        writer.write(formattedLines.toString());
+        writer.close();
+        FruitStorage.clearStock();
+    }
+
+    public Map<String, Integer> formatData(Map<FruitBatch, Integer> fruits) {
```

Comment:
this method shouldn't belong to this class 
it is one of the step working with data